scapy/utils.py

```diff
@@ -786,8 +796,12 @@ def read_block_epb(self, block, size):
 
 
 class PcapNgReader(RawPcapNgReader):
-    def __init__(self, filename):
-        RawPcapNgReader.__init__(self, filename)
+
+    alternative = PcapReader
+
+    def __init__(self, filename, fdesc, magic):
+        RawPcapNgReader.__init__(self, filename, fdesc, magic)
+
     def read_packet(self, size=MTU):
         rp = RawPcapNgReader.read_packet(self, size=size)
```

Comment:
If I understand your question well, this will not happen, since when `PcapNgReader()` is called against a regular PCAP file, a `PcapReader()` object is created:

```
$ file test.cap 
test.cap: tcpdump capture file (little-endian) - version 2.4 (Ethernet, capture length 65535)
$ ./run_scapy 
Welcome to Scapy (2.3.2-dev)
>>> PcapNgReader('/tmp/test.cap')
<scapy.utils.PcapReader object at 0x7fdedde78f90>
>>> _.read_packet()
<Ether  [...] type=IPv4 |<IP  version=4L ihl=5L [...] options=[] |<TCP [...] |<Raw  load='[...]' |>>>>
```